pkgs/top-level/python-packages.nix

```diff
@@ -18164,7 +18164,7 @@ in modules // {
   };
 
 
-  setuptoolsTrial = buildPythonPackage {
+  setuptoolsTrial = makeOverridable ({ twisted ? self.twisted }: buildPythonPackage {
     name = "setuptools-trial-0.5.12";
```

Comment:
A comment for that would get outdated quickly — I can add it, but I don't really see the point.